app/views/models/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="row row-cols-2">
+  <div class="col-9">
+    <div class="row row-cols-2 align-items-baseline">
+      <div class="col col-auto me-auto">
+        <h1><%= # @library.name
+        %></h1>
+      </div>
+      <div class="col col-auto ">
+        <a href="?order=name" class="btn <%= session["order"] == "name" ? "btn-primary" : "btn-light" %> btn-sm" title="sort by name"><i class="bi bi-book"></i></a>
+        <a href="?order=recent" class="btn <%= session["order"] == "recent" ? "btn-primary" : "btn-light" %> btn-sm" title="sort by time"><i class="bi bi-clock"></i></a>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="row row-cols-2">
+  <div class="col-9">
+    <div class="row row-cols-1 row-cols-md-2 row-cols-lg-3 mb-4">
+      <%= render partial: 'model', collection: @models %>
+    </div>
+    <% if current_user.pagination_settings["models"] %>
+      <%= paginate @models %>
+    <% end %>
+  </div>
+  <div class="col-3">
+
+    <%= card :secondary, "Actions" do %>
+      <%= link_to 'Delete Library', params[:library], method: :delete, data: {confirm: "Are you sure? Your files will not be affected."}, class: "btn btn-danger mb-3 me-3" if params[:library] %>
```

Comment:
This probably doesn't belong here any more, although it doesn't have anywhere else to go. Hmm. Maybe we just remove it for now, library deleting is maybe fine to just be in the advanced admin area.

Reply:
I was scratching my head as well.....

Reply:
I think just removing for now is fine.
```suggestion
```